test/playwright/playwright.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  globalSetup: "./setup/global.ts",
+  testDir: "./tests",
   browsers: [
```

Comment:
I don't think this browsers key is a playwright config options and can now be removed. It was only used with the jest setup to rename the test browser in each test's output (it's default output did not indicate electron at all despite the fact that it was driving our electron app)

Reply:
Awesome!